tests/Admin/ArticleCategoryAdminTest.php

```diff
@@ -0,0 +1,104 @@
+    }
+
+    public function testCreateCategoryWithoutCTASuccess()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([$prefix.'[name]' => 'Opinion', $prefix.'[slug]' => 'opinion', $prefix.'[position]' => 10]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_FOUND, $this->client);
+    }
+
+    public function testCreateCategoryWithCTASuccess()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([
+            $prefix.'[name]' => 'Opinion',
+            $prefix.'[slug]' => 'opinion',
+            $prefix.'[position]' => 10,
+            $prefix.'[ctaLink]' => 'http://www.google.fr',
+            $prefix.'[ctaLabel]' => 'Google link',
+        ]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_FOUND, $this->client);
+    }
+
+    public function testCreateCategoryWithCTAFail()
+    {
+        $this->authenticateAsAdmin($this->client, 'superadmin@en-marche-dev.fr', 'superadmin');
+
+        $crawler = $this->client->request('GET', $this->getUrl('admin_app_articlecategory_create'));
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $form = $crawler->selectButton('btn_create_and_edit')->form();
+
+        $prefix = $this->getPrefixAdminForm($form);
+        $form->setValues([
+            $prefix.'[name]' => 'Opinion',
+            $prefix.'[slug]' => 'opinion',
+            $prefix.'[position]' => 10,
+            $prefix.'[ctaLink]' => 'not a link',
+            $prefix.'[ctaLabel]' => 'Google link',
+        ]);
+        $this->client->submit($form);
+
+        $this->assertStatusCode(Response::HTTP_OK, $this->client);
+        $this->assertValidationErrors(['data.ctaLink'], $this->client->getContainer());
+    }
+
+    protected function getPrefixAdminForm(Form $form): ?string
+    {
+        foreach ($form->all() as $field) {
+            preg_match('/^(.*)\[.*\]$/', $field->getName(), $matches);
+            if ($matches) {
+                return $matches[1];
+            }
+        }
+
+        return null;
+    }
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->init([LoadAdminData::class]);
+    }
+}
```

Comment:
Please also implement `tearDown()` method to call `kill()` method and prevent memory leaks.